client/ipa-getkeytab.c

```diff
@@ -1083,8 +1091,10 @@ int main(int argc, const char *argv[])
                           "simultaneously.\n"));
         if (!quiet)
             poptPrintUsage(pc, stderr, 0);
+		poptFreeContext(pc);
```

Comment:
For consistency purposes please replace tab with white spaces

Reply:
The file is a bit of a mess of tabs and spaces unfortunately. I kept this one (I think) consistent with the lines around it.